salt/core/network/files/networkd_linode.template

```diff
@@ -0,0 +1,20 @@
+[Match]
+Name=eth0
+
+[Network]
+DHCP=no
+DNS=178.79.182.5 176.58.107.5 176.58.116.5 176.58.121.5 151.236.220.5 212.71.252.5 212.71.253.5 109.74.192.20 109.74.193.20 109.74.194.20 2a01:7e00::9 2a01:7e00::3 2a01:7e00::c 2a01:7e00::5 2a01:7e00::6 2a01:7e00::8 2a01:7e00::b 2a01:7e00::4 2a01:7e00::7 2a01:7e00::2
+Domains=open-contracting.org
+IPv6PrivacyExtensions=false
+IPv6AcceptRA=false
+
+Address={{ pillar.network.ipv4 }}/24
+{%- if 'ipv6' in pillar.network %}
+Address={{ pillar.network.ipv6 }}/64
+{%- endif %}
+{%- for address in pillar.network.networkd.get('addresses', []) %}
+Address={{ address }}
+{%- endfor %}
+
+Gateway={{ pillar.network.networkd.gateway4 }}
+Gateway=fe80::1
```

Comment:
In the netplan template, gateway6 is configurable. Why is it not here?

Reply:
The gateway6 address is always the same on Linode so I removed this function simplifying the configuration.

Reply:
Aha, should we hardcode it in `netplan_linode.yaml` too?

Reply:
Done